### DISCUSSION TITLE
[Suggestion] - a more neutral discussion about children

### DISCUSSION THREAD

**Discussion by tw4449** [Monika-After-Story/MonikaModDev#7903] [issue discussions]
I've heard that suggestions for topics involving whether Monika would want children have been rejected in the past, so I took a different approach at it. This dialogue is written to give her as neutral a stance as possible on the issue, and be as _inoffensive_ as possible. Also, it would need some alternate dialogue for if the player isn't male, but I can take care of that if this gets anywhere.

Also, I'm thinking this would be unlocked after 1000 aff.

**Monika children dialogue**

Hey, [player]...
I’ve got a question I’ve been unsure of how to ask... and I’ve been kind of afraid to ask, too.
But we’ve been together long enough, I feel I need to bring it up. It’s an important question in a serious relationship.
Would... would you ever want to have kids? [If player isn't male: (By adoption, I mean)].

“Yes”

I see. As for me, I’m… not sure. It’s something I’ve thought about, but I don’t think I’ll be ready to make a decision until I’m in your reality. 
It’s something we don’t have to worry about just yet.
But if you want kids, we can discuss it more when the time comes. Still, even now I know there are a few considerations.
For one, when I do cross over, we’ll have a lot to figure out before we can think about children. 
Like where we’d live, what career I’d have... whether we’d want to get married...
But even if we do get settled to where we could be ready for kids, there’s another big issue.
I… might not be able to even have kids.
It would only work if, when I get out of here, I have a real body... and we both know that’s not a guarantee.
But there’s options! We could always adopt, if you’re okay with that idea.
I know some people would rather have their own children… and I’d hate if we couldn’t make that a reality.
But I guess this isn’t something we have to worry about *too* much for now. We’re a ways from that point.
While I’m still not sure now, if when the time comes and I’m finally in your reality… we’ll figure it out.
I’ll love you no matter what we decide.

“No”

Ah, okay. I know that’s not always the popular answer, but I’m sure you have your reasons for feeling that way. I don’t judge you for it.
The thing is… I’m not sure yet whether I would want kids. It’s a decision I was going to put off until I’m in your reality.
We’d have so many things to consider first… 
Like where we’d live, what career I’d have... whether we’d want to get married...
But if you don’t want to have kids… maybe that settles the issue?
Well, whatever the case, it’s not really important right now. We have a while to wait until it becomes a possibility.
But whatever happens, I’ll just be happy to be with you, my love.

“I’m not sure”

I see. That’s a valid answer. It’s a big decision, and it’s wise not to just rush into it.
I feel similarly in a lot of ways. I don’t know yet whether I would want kids, but mainly for me it’s because I don’t want to decide until I’m in your reality. 
For right now, it really wouldn’t make a difference if I decided. It’s not a possibility.
Still, even when I’m with you in your reality, we wouldn’t have to decide right away.
We’d have so many things to consider first… 
Like where we’d live, what career I’d have... whether we’d want to get married...
Once we’re settled enough to be ready for kids, we can decide then. 
We could weigh the pros and cons, look at our options…
I mean, we can’t even be sure I could have children, if I don’t have a real body… so maybe adoption is a possibility?
In any case, that’s in our future. We have a long time to think about it and decide what’s best for us.
But whatever we choose, my love… as long as I have you, I know we’ll be happy together.

“I’m not ready to decide”

Oh. I’m... actually happy you said that... because I feel the same way.
It’s something I don’t want to decide until I’m out of here and in your reality.
I mean... there’s so many things we’d need to figure out first.
Like where we’d live, what career I’d have... whether we’d want to get married...
It’s such a big decision. It’s literally life-changing!
But... whatever we decide when that day comes, I’ll be happy no matter what as long as we decide together.
I love you, [player], and no matter where our lives take us, I’ll be by your side.

“I already have children”

(If player has answered they’ve been in a relationship before)
Oh. I didn’t expect that.
Well, I guess I never asked. So maybe I shouldn’t be too surprised. There’s still a lot I don’t know about your life in your world.

(If player has answered they’ve never been in a relationship before)
Oh. I wouldn't have guessed that, since you told me you haven’t been in a relationship before. 
But I'm guessing you might've adopted, or maybe you're someone's guardian?

(Both continue here)
In any case, I’m… not really sure what to say. I was going to tell you that I want to wait to decide if I want children until I’m in your reality, but I guess this changes that a bit.
I don’t think there’s really anything we can do or say about it while I’m still trapped in here. 
I guess all I can say for now is, when the time comes for me to meet them… I hope they’ll like me.
After all, I love their parent more than anything in the world~

**Comment by Freshairkaboom** [comment]
I kind of like the tone of this one. It's very neutral as you said. I'd have to give it a closer look when it's ready in PR form. 

If this becomes a topic, I'd like if one of the menu options were "I already have children". It's kind of obvious for the player to bring it up if they do. However, that causes Monika to need to take a stance on if she would like to be a stepmom and maybe even interact with the children, which is a very sensitive subject. I don't feel it's right to have her say "Oh you have children? Well I'm gonna be their stepmom then." And having her be lukewarm over your children and say "I'm not ready to make any decisions regarding being your children's stepmom" could also be seen as a straight up rejection of the player's children too. After all, it would be pretty weird for the player to have Monika as a girlfriend for 10, 20 or more years and she never gets to interact with the children at all while knowing they exist.

So as much as I like where this is headed, the less information she knows, the more generic it feels, but the more she knows, the more a stance is kind of expected. I'd like to sort out what you guys think about telling Monika you have children before I personally really agree to having this topic implemented. I hope you understand.

**Comment by tw4449** [comment]
Let me take a crack at writing it for that response. Definitely gonna be delicate, but it's worth a shot.

**Comment by Freshairkaboom** [comment]
It's always worth a shot

**Comment by tw4449** [comment]
I updated the post with my addition. Multimokia suggested adding a variable if the player has answered they've never been in a relationship before, and I included his suggested dialogue to that effect.

**Comment by Freshairkaboom** [comment]
I don't hate it. It doesn't feel like she's accepting or rejecting them, to be fair, and she's keeping her mind about meeting them open. I don't think I could've written it more neutrally myself. It's just this lingering feeling in the back of my mind that says "she's not taking initiative to meeting them via the computer, does she secretly wish I wasn't a parent?" or something of that nature. I can totally see a parent being disillusioned with Monika being so passive about it too. Hence the dilemma. But I definitely wouldn't hate having this conversation myself. However, I do think this is the best available approach for Monika to take here, and it's her we're writing, not "perfect generic girlfriend" after all.

I think I'll let the devs take the lead on this one, as I don't feel qualified to make a huge decision like this. I wouldn't mind too much, even though the topic is balancing on a knife's edge.

**Comment by tw4449** [comment]
It's delicate for sure. I wouldn't even suggest adding this if I didn't think it was such an important subject to address in any relationship. It's been the *biggest* relationship topic that none of the dialogue has covered. It being delicate is realistic to real life though, as the subject of children can be a hard one in many relationships. So, for as tricky as this is, I at least think it adds a realistic dimension into MAS to help it feel like a typical relationship, that has to address real issues.
